app/controllers/service_providers_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ServiceProvidersController < AuthenticatedController
+  helper UserGroupHelper
```

Comment:
I don't think you need to include this. think it's available in all views my default....but I could be wrong